.github/workflows/master.yaml

```diff
@@ -39,20 +39,17 @@ jobs:
         run: |
           minikube start --wait all
           minikube docker-env | sed '/^#/d;s/="\(.*\)"/=\1/g;s/export //g' >> $GITHUB_ENV
-      - name: Cache Go Modules
+      - name: Go Cache
         uses: actions/cache@v1
         with:
-          path: ~/go/pkg/mod
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            ~/Library/Caches/go-build
+            %LocalAppData%\go-build
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
```

Comment:
The hash of `go.sum` makes sense as a cache key for `GOMODCACHE` but do want both the module and build cache keyed off it? I agree that `hashFiles('**.go')` (removed below) is effectively worthless since the number of commits not touching Go files is basically 0, but should the build cache be keyed of something like the Go version?